src/main/scala/mesosphere/marathon/state/PathId.scala

```diff
@@ -97,10 +118,46 @@ case class PathId(path: Seq[String], absolute: Boolean = true) extends Ordered[P
   override val hashCode: Int = scala.util.hashing.MurmurHash3.productHash(this)
 }
 
+case class AbsolutePathId(path: Seq[String]) extends PathId {
+  override val absolute: Boolean = true
+  override def asAbsolutePath: AbsolutePathId = this
+  protected def toString(delimiter: String): String =
+    path.mkString("/", delimiter, "")
+}
+
+object AbsolutePathId {
+  /**
+    * Parse the string as a path, but interpret it as relative to root, if it is relative.
```

Comment:
Could you give some examples in the docs? Eg `/dev/prod` vs `dev/prod` vs `../prod`.